packages/alfa-dom/src/jsx.ts

```diff
@@ -24,16 +24,16 @@ export function jsx<N extends string = string>(
 
     switch (name) {
       case "style":
-        const properties = value as Record<string, string>;
-
-        for (const [name, value] of entries(properties)) {
-          style[name] = value;
```

Comment:
Due to [the way `unknown` is now handled](https://devblogs.microsoft.com/typescript/announcing-typescript-4-8/#improved-intersection-reduction-union-compatibility-and-narrowing), the non-null checks at line 21 already narrowed `value` to an object. So there is no need for the type assertion anymore.